<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>suppressionRulesGUIDlg</class>
 <widget class="QWidget" name="suppressionRulesGUIDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>379</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>System / Non-physical Subsystem Alarm Inhibitions</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="CtlTableWidget" name="tblAlarmRules">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount">
      <number>7</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Removable</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date / Time</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Asset Name</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Alarm Type</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Operator</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="CtlButton" name="btnRemoveInhibition">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Remove Inhibition</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="CtlButton" name="btnPrintList">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Print List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CtlButton" name="btnExportList">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Export List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CtlButton" name="btnRefresh">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CtlButton" name="btnClose">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="CtlGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="CtlRadioButton" name="rdoShowAll">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Show All Inhibitions</string>
      </property>
     </widget>
     <widget class="CtlRadioButton" name="rdoFilterLocation">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>221</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter on Location :</string>
      </property>
     </widget>
     <widget class="CtlComboBox" name="cboLocation">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>311</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CtlButton</class>
   <extends>QPushButton</extends>
   <header>bus/common_control/src/button/CtlButton.h</header>
  </customwidget>
  <customwidget>
   <class>CtlRadioButton</class>
   <extends>QRadioButton</extends>
   <header>bus/common_control/src/button/CtlRadioButton.h</header>
  </customwidget>
  <customwidget>
   <class>CtlGroupBox</class>
   <extends>QGroupBox</extends>
   <header>bus/common_control/src/container/CtlGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CtlComboBox</class>
   <extends>QComboBox</extends>
   <header>bus/common_control/src/input/CtlComboBox.h</header>
  </customwidget>
  <customwidget>
    <class>CtlTableWidget</class>
    <extends>QTableWidget</extends>
    <header>bus/common_control/src/itemview/CtlTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
